daliuge-engine/dlg/drop.py

```diff
@@ -1721,18 +1721,18 @@ class PlasmaDROP(AbstractDROP):
     '''
     A DROP that points to data stored in a Plasma Store
     '''
-    object_id = dlg_string_param('object_id', plasma.ObjectID.from_random().binary())
+    object_id = dlg_string_param('object_id', None)
     plasma_link = dlg_string_param('plasma_link', '/tmp/plasma')
 
     def initialize(self, **kwargs):
        pass
```

Comment:
Now that `object_id` defaults to `None` I was suggesting you could initialise `self.object_id` here (if `None`) from the `self.uid` value. But ObjectIDs are 20 bytes long, so you need to make sure you collapse `self.uid` into that size (via hashing, substring, suffix, prefix, or any other mechanism). Or just use a random value; in any case the main point before was not using the same random Object ID as the default value for all instances.